rom/dev/tests/rom_integration_tests/test_idevid_derivation.rs

```diff
@@ -108,15 +113,60 @@ fn test_generate_csr_stress() {
                 fuses.uds_seed
             )
         });
-        let pub_key = req.public_key().unwrap();
+        let idevid_pubkey = req.public_key().unwrap();
         assert!(
-            req.verify(&pub_key).unwrap(),
+            req.verify(&idevid_pubkey).unwrap(),
             "Invalid public key. Unable to verify CSR with UDS seed {:?}",
             fuses.uds_seed
         );
+
+        let ldev_cert = get_and_verify_cert(
+            &mut hw,
+            u32::from(CommandId::TEST_ONLY_GET_LDEV_CERT),
+            &idevid_pubkey,
+            &fuses.uds_seed,
+        );
+        let _fmc_cert = get_and_verify_cert(
+            &mut hw,
+            u32::from(CommandId::TEST_ONLY_GET_FMC_ALIAS_CERT),
+            &ldev_cert.public_key().unwrap(),
+            &fuses.uds_seed,
+        );
     }
 }
 
+fn get_and_verify_cert(
+    hw: &mut DefaultHwModel,
+    cmd_id: u32,
+    pubkey: &PKey<Public>,
+    uds: &[u32; 12],
```

Comment:
super nit: Can you call this `test_uds`. Just makes the test look less scary to the casual observer 😁